### ISSUE TITLE
Disabled Select for language when editing in the front end

### ISSUE THREAD

**Issue by brianteeman** [joomla/joomla-cms#46702]
### Steps to reproduce the issue
1. On a multilingual website open an article for editing in the front end
2. Using a screenreader try to "read" the language of the article
3. Alternatively try to focus on the language field using the keyboard

### Expected result
The language of the article is announced by the screenreader

### Actual result
As the field is disabled the screenreader never "sees" the field

### Comments
The language select is rendered as disabled and a hidden input contains the language value

I dont know why you cant change the language of an article on the frontend but its clearly by design. However instead of a a disabled select that you cannot get the value of I would instead have expected this to be a regular input field marked as readonly

### Device and system info

* What version of Joomla were you using?
All

* What device were you using? (eg, phone, laptop)
All

* What web browser were you using? (eg, Chrome, Safari)
Chrome

* If you were using assistive technology, please let us know the device or software you were using (eg VoiceOver, JAWS, sip-and-puff device, etc)

Narrator, keyboard navigation

### Additional comments

If you know which [WCAG](https://www.w3.org/TR/WCAG/) or [ATAG](https://www.w3.org/TR/ATAG20/) criteria this violates, please indicate it here:

Lots of WCAG Level A criteria

**Comment by brianteeman** [comment]
weird - it looks like it might have been a problem on a specific test site as i cant replicate it now. will close this for now and re-open if I can replicate
